python/lsst/obs/hsc/hscMapper.py

```diff
@@ -164,13 +165,13 @@ def _flipChipsLR(exp, wcs, dataId, dims=None):
         return exp, wcs
 
     def std_raw_md(self, md, dataId):
-        if False:            # no std_raw_md in baseclass
-            md = super(HscMapper, self).std_raw_md(md, dataId)  # not present in baseclass
-        #
-        # We need to flip the WCS defined by the metadata in case anyone ever constructs a Wcs from it
-        #
+        """We need to flip the WCS defined by the metadata in case anyone ever
+        constructs a Wcs from it.
+        """
         wcs = afwGeom.makeSkyWcs(md)
-        wcs = self._flipChipsLR(None, wcs, dataId, dims=afwImage.bboxFromMetadata(md).getDimensions())[1]
+        wcs = self._flipChipsLR(None, wcs, dataId['ccd'],
+                                dims=afwImage.bboxFromMetadata(md).getDimensions())[1]
+        # NOTE: we don't know where the 0.992 magic constant came from. It was copied over from hscSimMapper.
```

Comment:
That's not terrifying at all 😱

Reply:
Ah, that reminds me.  @parejkoj, have a look at my adjustments to this commit here: https://github.com/lsst/obs_subaru/commit/95349380f894453f08bab564d4b70991880fd2d5

Reply:
@laurenam ah, sure, that's at least a better explanation. I'll make sure and be careful when rebasing on top of your work before I merge.